cfpb/cfgov-refresh
# Fix our HowTo schema markup support per newest guidance

At some point since we first implemented it, Google made the `name` field required for the `HowTo` itself (as opposed to just the individual `HowToStep`s). This commit adds a new corresponding `title` field to the `HowTo` block in Wagtail, and outputs that in the template.

Also includes some other minor cleanup of that template, including putting the steps inside an ordered list.

/cc @kelleyholden 


## Additions

- `title` field added to `HowTo` block in Wagtail, output as the `name` field at the top level of the `HowTo` in the markup

## Changes

- `HowToStep`s are now list items of an ordered list, to provide for automatic numbering, which is useful when following the latest guidance that the step number should not be included in the step's own `name` field.


## Testing

1. Grab a fresh database dump
1. Pull branch
1. `./cfgov/manage.py migrate`
1. `gulp styles`
1. Edit http://localhost:8000/admin/pages/13816/edit/ to see the new field. Move the existing H2 content in there to accommodate it. Remove the numbers in the step titles. Confirm that you end up with something like the below screenshot.


## Screenshots

How a complete HowTo would now be rendered:

![image](https://user-images.githubusercontent.com/1044670/81118266-9c15e100-8ef6-11ea-84e5-7d61186acac2.png)


## Notes

- See Google's latest HowTo guidance at: https://developers.google.com/search/docs/data-types/how-to


## Checklist


## Testing checklist

### Browsers
